npc/re/mobs/dungeons/tur_dun.txt

```diff
@@ -51,3 +52,43 @@ tur_dun04,99,93,20,20	boss_monster	Turtle General	1312,1,3600000,600000,0
 tur_dun05,0,0	monster	Freezer	1319,7,5000
 tur_dun05,0,0	monster	Permeter	1314,4,5000
 tur_dun05,0,0	monster	Assaulter	1315,1,5000
+
+//==================================================
+// Illusion of Turtle
+//==================================================
+tur_d03_i	monster	Ominous Permeter	3800,15
+tur_d03_i	monster	Ominous Freezer	3801,40
+tur_d03_i	monster	Ominous Assulter	3799,35
+tur_d03_i	monster	Ominous Solider	3802,40
+tur_d03_i	monster	Ominous Heater	3803,10
+
+tur_d04_i	monster	Ominous Permeter	3800,4,5000,0,"illusion_turtle_boss::OnEvent"
+tur_d04_i	monster	Ominous Freezer	3801,15,5000,0,"illusion_turtle_boss::OnEvent"
+tur_d04_i	monster	Ominous Assulter	3799,42,5000,0,"illusion_turtle_boss::OnEvent"
+tur_d04_i	monster	Ominous Solider	3802,1,5000,0,"illusion_turtle_boss::OnEvent"
+tur_d04_i	monster	Ominous Heater	3803,33,5000,0,"illusion_turtle_boss::OnEvent"
+
+// boss spawn
+-	script	illusion_turtle_boss	-1,{
+	end;
+OnEvent:
+	if (.boss)
+		end;
+	.count++;
+	if (.count < 1000)	// todo unknown amount
+		end;
+	.boss = 1;
+	// mapannounce "tur_d04_i", "", bc_map, "0x70dbdb";	// todo unknown announce
+	monster "tur_d04_i",0,0,"Ominous Turtle General",3804,1, "illusion_turtle_boss::OnBoss";	// todo unknown location
```

Comment:
On iRo (dont know kRo, never spawn it before there) the location of the mvp spawn should be "next to the last turtle killed"